inductiva/resources/machines.py

```diff
@@ -173,6 +177,12 @@ def from_api_response(cls, resp: dict):
         machine_group.num_active_machines = int(resp["num_vms"])
         return machine_group
 
+    def current_machines_to_str(self) -> str:
+        """Returns a string representation of the 
+        number of machines currently running.
+        """
+        return f"{self.num_active_machines}/{self.max_machines}"
```

Comment:
maybe give some feedback that the request refers to the max number of machines:

```suggestion
        return f"{self.num_active_machines}/{self.max_machines} (max)"
```